_data/simple-icons.json

```diff
@@ -7423,6 +7423,18 @@
             "hex": "FF8800",
             "source": "https://help.micro.blog"
         },
+        {
+            "title": "Microchip Technology Incorporated",
```

Comment:
```suggestion
            "title": "Microchip",
```

Reply:
The `Microchip` is enough. There is no need to use the company's full name.

And those aliaes can be removed.

Reply:
I have fixed the JSON and SVG to match your suggestion.